### PR TITLE
even more odd photos hint

### PR DESCRIPTION
**Pull request by SansPapyrus683**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.

### CODE REVIEW

**Comment by SansPapyrus683** [comment]
@envyaims @ryanchou-dev

**Review by ryanchou-dev — APPROVED** [review]
nice

File: solutions/bronze/usaco-1084.mdx [hunk 1]
```diff
@@ -163,7 +163,7 @@ import java.util.*;
 public class OddPhotos {
 	public static void main(String[] args) throws IOException {
 		BufferedReader in =
-			new BufferedReader(new InputStreamReader(System.in));
+		    new BufferedReader(new InputStreamReader(System.in));
 		int odd = 0;
```
Comment (SansPapyrus683) [inline solutions/bronze/usaco-1084.mdx]: WHY IS IT USING SPACES HERE WHAT THE HECK

Comment (bqi343) [reply solutions/bronze/usaco-1084.mdx]: tabs are used for indentation only

**Review by envyaims — APPROVED** [review]
👍

**Pull request merged**